### PR TITLE
ARROW-14626: [Docs][Ruby][Release] Update sites to indicate versions tested on

### PR DESCRIPTION
**Pull request by bkmgit**

### CODE REVIEW

File: ruby/red-arrow-cuda/red-arrow-cuda.gemspec [hunk 1]
```diff
@@ -48,4 +48,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("bundler")
   spec.add_development_dependency("rake")
   spec.add_development_dependency("test-unit")
+  spec.required_ruby_version = Gem::Requirement.new("=> 2.6")
```
Comment (kou) [inline ruby/red-arrow-cuda/red-arrow-cuda.gemspec:51]: This is a invalid syntax. `>=` is correct:

```suggestion
  spec.required_ruby_version = Gem::Requirement.new(">= 2.6")
```

**Review by kou — COMMENTED** [review]
I don't want to specify this. Because the current implementation works with Ruby 2.5. (The current our tests require Ruby 2.6 or later but we can support Ruby 2.5 with a small change.)

BTW, we should not reuse ARROW-14626 for this. This isn't for Website related change.

I close this for now.

File: ruby/red-arrow-cuda/red-arrow-cuda.gemspec (hunk 1 above)
Comment (bkmgit) [reply ruby/red-arrow-cuda/red-arrow-cuda.gemspec:51]: @kou  Ok, will create new issue and add support for 2.5.  Is this a priority for 6.0.1 ?

Comment (kou) [reply ruby/red-arrow-cuda/red-arrow-cuda.gemspec:51]: It's not needed for 6.0.1 because this is only test issue.

**Pull request closed without merging**